v2.0/attributes/marker.yml

```diff
@@ -1,7 +1,10 @@
 ---
 name: marker
 description: |-
-  The position marker at which to begin the response.
+  Defines the position marker at which to begin returning results. This is
+  used when paginating using marker-based pagination.
+
+  This requires `usemarker` to be set to `true`.
```

Comment:
Is this always true? I thought `usemarker` is required only for the APIs where offset-based pagination is the default.

Reply:
We only have 1 API endpoint that only supports marker based pagination, and in that endpoint we don't re-use this bit of copy.